Announce.php

```diff
@@ -103,4 +105,143 @@ function schema() {
 			array( 'UpdateFunction', 'delete_orphans_dismissals' ),
 		);
 	}
+
+	/**
+	 * Add the RESTful routes handled by this plugin.
+	 *
+	 * @param string $p_event_name The event name
+	 * @param array  $p_event_args The event arguments
+	 * @return void
+	 */
+	public function routes( $p_event_name, $p_event_args ) {
+		$t_app = $p_event_args['app'];
+		$t_plugin = $this;
+		$t_app->group(
+			plugin_route_group(),
+			function() use ( $t_app, $t_plugin ) {
+				$t_app->post( '/dismiss/{context_id}', [$t_plugin, 'route_dismiss'] );
+				$t_app->get( '/context/{message_id}', [$t_plugin, 'route_new_context'] );
```

Comment:
The name of the method indicates that you are creating a context rather than getting info/form for creating a new context.  I would rename the method.

Reply:
Makes sense, will do